### ISSUE TITLE
Report on which languages an application were translated into

### ISSUE THREAD

**Issue by Miguel-314** [civiform/civiform#5393]
**Is your feature request related to a problem? Please describe.**
Some but not all programs ask applicants what their preferred language is. However, its possible that a applicant chooses to auto translate the form into once language and then selects a different language as their "preferred" language. This report could also provide meaningful data on which languages are the most used languages on CiviForm. Program specific data could be used to inform future outreach and marketing efforts. 

**Describe the solution you'd like**
A report that shows total applications translated and which languages were selected. It would also be more helpful if this report was able to pull data from universal "preferred language" questions when they are used as part of an application. 

**Additional context**

**Done when**
A report is created or other suggestions on how to visualize this data.

**Comment by msprenke** [comment]
This information is currently surfaced in the program admin report and could be used for analysis there.

It could be surfaced in the program admin view.  @Miguel-314 could you get a feel for how many programs would like this?

**Comment by Miguel-314** [comment]
So far SHA and DEEL have expressed interest in this. It could be as easy as pulling a monthly report and emailing it out to program staff upon request or as part of a centralize monthly report that includes additional lessons learned from user testing and user feedback.

**Comment by Miguel-314** [comment]
Last week at Governance (8.14.24) there was conversation about more robust reporting features in CiviForm. Or better connections to existing reporting tools like Power BI, Tableau, sharepoint, that might be able to handle more robust reporting requests. If this becomes a larger epic this issue should be added to that epic.

**Comment by Miguel-Seattle** [comment]
I don't believe this is necessary anymore since we have been pulling this data from pivot tables and manually compiling it for City of Seattle reporting.
